saleor/graphql/product/tests/test_variant.py

```diff
@@ -607,7 +591,7 @@ def test_get_product_variant_preorder_as_customer_allowed_fields(
 
 @patch("saleor.plugins.manager.PluginsManager.product_variant_created")
 @patch("saleor.plugins.manager.PluginsManager.product_variant_updated")
-def test_create_variant(
+def test_create_variant_with_name(
```

Comment:
Do we have a similar test for `class ProductVariantBulkCreate(BaseMutation):`?

Reply:
noo, I will add this